src/App/Components/Header.js

```diff
@@ -11,6 +11,7 @@ class Header extends React.Component {
 		this.setState({ show: !currentState });
 	};
 	render() {
+		const { showSideNav } = this.props;
 		return (
 			<>
 				<html lang='en' />
```

Comment:
@lemoissonM  great job but remove this because it is in the `Public HTML file` and of cause <>